ppl/zqd/recruiter/README.md

```diff
@@ -0,0 +1,82 @@
+# zqd recruiter
+
```

Comment:
@marktwallace : I filed https://github.com/brimsec/zq/issues/1755 to capture the changes we've discussed a couple of times on the protocol.